os/CMakeLists.txt

```diff
@@ -3,9 +3,10 @@
 # voucher_private.h are included in the source tarball
 
 install(FILES
-          object.h
+          generic_base.h
```

Comment:
Should the unix and windows base.h versions just import generic_base.h?

Reply:
I assume this new header needs to be installed to be importable at the expected install location.